### PR TITLE
Lower frequency of bee memes

### PR DESCRIPTION
**Pull request by cblanck**

### CODE REVIEW

File: scripts/memes.coffee [hunk 1]
```diff
@@ -20,6 +20,10 @@ module.exports = (robot) ->
   robot.hear /\bbaber'?s\s?way\b/i, (msg) ->
     msg.send 'http://i.imgur.com/rIC8Vuc.gif'
 
+  robot.hear /bee+s?\b/i, (message) ->
+    if Math.random() <= 0.05
```
Comment (adewinter) [inline scripts/memes.coffee]: ONLY 5% 💢 💢 :rage1:

Comment (adewinter) [reply scripts/memes.coffee]: Make it 25% and I'd be down

Comment (ethowitz) [reply scripts/memes.coffee]: Drive-by: have you considered making this 1.0?

Comment (aripollak) [reply scripts/memes.coffee]: :clap:

Comment (xxv) [reply scripts/memes.coffee]: I'm down with the 5%. Add some excitement!

**Pull request closed without merging**